google/cloud/storage/_helpers.py

```diff
@@ -134,6 +136,14 @@ def _query_params(self):
             params["userProject"] = self.user_project
         return params
 
+
+    def _call_api(self, client, retry, **kwargs):
+        call = functools.partial(client._connection.api_request, **kwargs)
```

Comment:
Is this because the interface for the retry function is not expected to pass in any arguments?

Reply:
Copied it over mainly:
At the moment no, I'm not making it configurable for the first pass and only introducing it to help support/unblock customer friction when using the library.